src/PrestaShopBundle/Install/Install.php

```diff
@@ -1026,6 +1026,10 @@ protected function executeAction(array $modules, string $action, string $errorMe
         foreach ($modules as $module_name) {
             $moduleException = null;
 
+            if ($moduleManager->isInstalled($module_name)) {
```

Comment:
If the `action` is `postInstall`, then the action will not be executed;

Reply:
Nice catch!